x64/Release/ZERO_CHECK/ZERO_CHECK.log

```diff
@@ -0,0 +1,14 @@
+﻿  Checking Build System
```

Comment:
That file slipped in too! Maybe add the `x64/` subfolder to .gitignore

Reply:
Add "/X64/" to .gitignore

Reply:
Don't forget to remove the actual file! I can't merge these unwanted files.